src/Parallel/Algorithm.hpp

```diff
@@ -354,7 +354,11 @@ class AlgorithmImpl<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>> {
 
   bool terminate_{true};
 
-  using all_cache_tags = get_const_global_cache_tags<metavariables>;
+  using const_cache_tags = get_const_global_cache_tags<metavariables>;
```

Comment:
We actually don't want to be able to get mutable_cache_tags from the DataBox as you need to check if they are valid.  So this change should be dropped unless I am missing some use of the new aliases, in which case only the `const_cache_tags` should be passed into `db::wrap_tags_in<Tags::FromGlobalCache, all_cache_tags>` in the two places it occurs

Reply:
I did this because the compute tag CoordinatesMeshVelocityAndJacobiansCompute takes FunctionsOfTime as an argument tag. Disallowing mutable cache tags to be accessible from the databox means that tag can no longer be used as a compute tag, and updating its value will have to be explicitly done as its own action or a part of another action. I deliberately didn't want to do this since it changed more existing code, but if you think that's how this tag should be handled then that change will have to be made.

Reply:
I discussed this with @nilsdeppe and we still don't want to be able to get mutable cache tags from the DataBox as there is no mechanism for checking if they are in a valid state.  Therefore, like you say, any compute tag that takes FunctionsOfTime will need to be replaced with a simple tag that needs to be mutated in a separate action or part of another action.  For now the simplest way is probably to do so in its own action that would be called right after the time is updated.  If you want to discuss this in further detail, please let us know and we would be happy to hop into a hangout.

Reply:
This was discussed again last week at a meeting that included @nilsdeppe @wthrowe @osheamonn @markscheel @moxcodes @fmahebert.  There we decided that we do want `FunctionsOfTime` accessible through the `DataBox`, but instead of adding all mutable cache tags, for the time being we will only add the `FunctionsOfTime`.